include/cantera/oneD/Sim1D.h

```diff
@@ -142,15 +142,22 @@ class Sim1D : public OneDim
               const std::string& desc, int loglevel);
 
     /**
-     * Save the current solution to a container file.
+     * Save the current solution to a container or CSV file.
+     *
+     * For HDF and YAML, the entire content of the object is saved; for CSV, only the
+     * main 1D domain is saved.
+     *
      * @param fname  Name of output container file
      * @param id  Identifier of solution within the container file
      * @param desc  Description of the solution
      * @param overwrite  Force overwrite if name exists; optional (default=false)
      * @param compression  Compression level (optional; HDF only)
+     * @param basis  Output mass ("Y"/"mass") or mole ("X"/"mole") fractions (CSV only);
+     *      if omitted (default=""), the native storage mode is used
```

Comment:
Can you clarify what the native storage mechanism means? Does it mean the basis set in the underlying object?

Reply:
Correct - will update the docstrings.